src/multivariate/powerspherical.jl

```diff
@@ -0,0 +1,77 @@
+"""
+The Power Spherical distribution is useful as a replacement for the von Mises-Fisher distribution.
+
+The probability density function of the Power Spherical distribution parameterized by the mean direction ``\\mu`` and the concentration parameter ``\\kappa`` is given by
+    
+```math
+p_{X}(x ; \\mu, \\kappa)= \\left\\{2^{\\alpha+\\beta} \\pi^{\\beta} \\frac{\\Gamma(\\alpha)}{\\Gamma(\\alpha+\\beta)}\\right\\}^{-1}\\left(1+\\mu^{\\top} x\\right)^{\\kappa}
+```
```

Comment:
Can you make it consistent with other docstrings, add signatures etc. and add it to the docs?